src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1011,26 +1011,90 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 			$body[] = "\n";
 			$body[] = __( 'Please check your site now. It’s possible that everything is working. If there are updates available, you should update.' );
 			$body[] = "\n";
-
+            
+            // Check for failed themes and plugins listing with failed_update_plugins_themes option_name
+			$failed_update_plugins_themes = (array) get_site_option( 'failed_update_plugins_themes', array() );
+			
 			// List failed plugin updates.
 			if ( ! empty( $failed_updates['plugin'] ) ) {
 				$body[] = __( 'These plugins failed to update:' );
 
 				foreach ( $failed_updates['plugin'] as $item ) {
 					$body[] = "- {$item->name}";
+                    
+                // Failed Plugin list to store into the database 
+                $failed_plugin = array( 'name' => $item->name, 'failure_time' => time(), 'file_name' => $item->item->plugin );
+                if( isset( $failed_update_plugins_themes['plugin'] ) && ! empty( $failed_update_plugins_themes['plugin'] ) ) {
+                    if( array_search( $item->item->plugin, array_column( $failed_update_plugins_themes['plugin'], 'file_name' ) ) === false ) {
+                        $failed_update_plugins_themes['plugin'][] = $failed_plugin;	
+                    }	
+                } else {
+                    $failed_update_plugins_themes['plugin'][] = $failed_plugin;
+                }
+                    
 				}
 				$body[] = "\n";
 			}
+            
+            
+            // Check for 3 days passed or not
+			if( isset( $failed_update_plugins_themes['plugin'] ) && ! empty( $failed_update_plugins_themes['plugin'] ) ) {
+				foreach( $failed_update_plugins_themes['plugin'] as $key => $failed_plugin ) {
+					if( isset( $failed_plugin ) && ! empty( $failed_plugin['failure_time'] ) ) {
+						$current_time 	= time(); 
+						$lastcheck  	= $failed_plugin['failure_time'];
+						$datediff   	= $current_time - $lastcheck;
+						
+						$days_check = round( $datediff / ( 60 * 60 * 24 ) );
+						if( $days_check >= 3 ) {
+							unset( $failed_update_plugins_themes['plugin'][$key] );
+							$body[] = $failed_plugin['name'];
+						}			
+					}	
+				}	
+			}
 
 			// List failed theme updates.
 			if ( ! empty( $failed_updates['theme'] ) ) {
 				$body[] = __( 'These themes failed to update:' );
 
 				foreach ( $failed_updates['theme'] as $item ) {
 					$body[] = "- {$item->name}";
+                    
+				    // Failed Plugin list store in database
```

Comment:
Same comment as above: `$past_failures[ $item->item->theme ] => time();` should replace this whole block, and location is incorrect.